scripts/versions.ts

```diff
@@ -2,14 +2,14 @@ import path from 'node:path';
 import themes from '../themes.json';
 import { readFileSync, writeFileSync } from 'node:fs';
 
-const root = path.join(__dirname, '..');
+export const root = path.join(__dirname, '..');
 const versionsFileName = path.join(root, 'versions.json');
 
-function readJson(filename: string): { [key: string]: any } {
+export function readJson(filename: string): { [key: string]: any } {
   return JSON.parse(readFileSync(filename, { encoding: 'utf8' }));
 }
 
-function writeJson(filename: string, json: { [key: string]: any }) {
+export function writeJson(filename: string, json: { [key: string]: any }) {
```

Comment:
Thinking about how to add a new line to just the end of the package.json files if you didn't want to update how `versions.json` is formatted.

You could update the function like so
```suggestion
export function writeJson(filename: string, json: { [key: string]: any }, addNewLine?: boolean) {
  const stringifiedJson = JSON.stringify(json, null, 2);
  const content = addNewLine ? `${stringifiedJson}\n` : stringifiedJson
  writeFileSync(filename, content);
}
```

and then where `writeJson` is invoked in the `bump-child-deps-utils`, you could call it like `writeJson(jsonFileLocation, updatedJson, true);`

Reply:
Love it. Thank you @kim-cmsds